app/views/scan_analytics/index.html.haml

```diff
@@ -0,0 +1 @@
+= render partial: 'scan_analytics/index'
```

Comment:
Configure your editor to automatically add newlines. If you are not sure, just open this file in vim and then type in `:x` to save and close the file. Vim will automatically add the newline for you.